src/cli/cli.cpp

```diff
@@ -4234,12 +4234,41 @@ template <> otError Interpreter::Process<Cmd("partitionid")>(Arg aArgs[])
 {
     otError error = OT_ERROR_INVALID_COMMAND;
 
+    /**
+     * @cli partitionid
+     * @code
+     * partitionid
+     * 4294967295
+     * Done
+     * @endcode
+     * @par api_copy
+     * #otThreadGetPartitionId
+     * @par
+     * Get the Thread Network Partition ID.
+     */
     if (aArgs[0].IsEmpty())
     {
         OutputLine("%lu", ToUlong(otThreadGetPartitionId(GetInstancePtr())));
         error = OT_ERROR_NONE;
     }
 #if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+    /**
+     * @cli partitionid preferred (partitionid) (get,set)
```

Comment:
Let's remove the (partitionid) from the name

`* @cli partitionid preferred (get,set)`